src/microbe_stage/systems/ProcessSystem.cs

```diff
@@ -790,7 +799,7 @@ private void ProcessNode(ref BioProcesses processor, ref CompoundStorage storage
             if (entry.Key.IsEnvironmental)
                 continue;
 
-            var inputRemoved = entry.Value * totalModifier;
+            var inputRemoved = entry.Value * totalModifier * process.SpeedMultiplier;
```

Comment:
I think applying the speed multiplier to `totalModifier` on line 775 would be the most error-proof way as I still see that this doesn't apply to the outputs, so now inputs are consumed less than outputs are added.